src/py42/modules/detectionlists.py

```diff
@@ -5,3 +5,37 @@ def __init__(self, microservice_client_factory):
     @property
     def departing_employee(self):
         return self._microservice_client_factory.get_departing_employee_client()
+
+    @property
+    def high_risk_employee(self):
+        return self._microservice_client_factory.get_high_risk_employee_client()
+
+    def update_notes(self, user_id, notes):
+        # Need to figure out how detection_list_user_client will be instantiated
+        self._detection_list_user_client.update_notes(user_id, notes)
+
+    def add_risk_tag(self, user_id, tags):
+        # Below note applies to `remove_risk_tag`, `add_cloud_alias`, `remove_cloud_alias` as well.
+
+        # Changed method signature, used 'tags' instead of 'tag' as compared to requirement,
+        # as the API expects a list.
+
+        # Suggestion:
+        # we can allow user to pass either of string or list and manage it on our side as
+        # below, i.e if its not a list, make a list and pass it to the API.
+        # if type(tags) is str:
+        #   tags = [tags]
+
+        # I don't see much validation done in other modules as well, do we need to add
```

Comment:
In this case i think it would be helpful to throw an error. We can re-evaluate other modules on an as-needed basis but I'm trying to keep the code simple and in many cases we just allow the user to fail.

Reply:
API returns bad request error, same as earlier comment.